src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php

```diff
@@ -225,8 +227,13 @@ public function dump(array $options = [])
 
         $proxyClasses = $this->inlineFactories ? $this->generateProxyClasses() : null;
 
+        if ($options['preload_classes']) {
+            $this->preload = $options['preload_classes'];
```

Comment:
I would not assign it here if it needs to be combined.

Reply:
updated